keras_app/views/layers_export.py

```diff
@@ -570,6 +571,24 @@ def batch_norm(layer, layer_in, layerId, idNext, nextLayer,):
     return out
 
 
+def bidirectional(layerId, idNext, net, layer_in, layer_map):
+    if net[layerId]['params']['merge_mode'] == '':
+        net[layerId]['params']['merge_mode'] = None
+    mode = net[layerId]['params']['merge_mode']
+    out[layerId] = Bidirectional(
+        layer_map[net[idNext]['info']['type']](net[idNext], layer_in, idNext),
+        mode)
+    return out
+
+
+def time_distributed(layerId, idNext, net, layer_in, layer_map):
+    mode = net[layerId]['params']['merge_mode']
```

Comment:
This doesn't have a mode parameter

Reply:
... Yeah, I think I may have accidentally copy-pasted that. Thanks for pointing it out.